Saule/Http/JsonApiMediaTypeFormatter.cs

```diff
@@ -155,10 +155,10 @@ public override bool CanWriteType(Type type)
 
         private async Task WriteJsonToStream(JToken json, Stream stream)
         {
-            using (var writer = new StreamWriter(stream, Encoding.UTF8, 2048, true))
+            using (var writer = new StreamWriter(stream, new UTF8Encoding(false), 2048, true))
```

Comment:
Can you add a named parameter for this boolean? I think it will make the code just a little easier to understand.

Reply:
Good call. Done 😄